src/components/StatueRoutes.js

```diff
@@ -6,7 +6,8 @@ import { BrowserRouter as Route, Switch } from "react-router-dom";
 export class StatueRoutes extends Component {
   render() {
     const newRoute = SlaverStatueList.map((statue, index) => {
-      const url = `/StatueDetailContainer/${statue.name.replace(/\s/g, "-")}`;
+      const slug = `${statue.name.replace(/\s/g, "-")}-${statue.statueId}`;
```

Comment:
Again, I'd prefer:
```
const { name, statueId } = statue
```

Also if it's in an object called `statue`, `statueId` should just be `id`.